index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+
```

Comment:
- [ ] Please includes an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

![image](https://user-images.githubusercontent.com/74744787/160407684-b92d2cde-23f6-43e7-89fb-2022cec3eedc.png)

you can include this as a reason : 
Meta tags are important because they impact how your site appears in the SERPs and how many people will be inclined to click through to your website. They will therefore impact your traffic and engagement rates, which can impact your SEO and rankings.